<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,15">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>串口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MyComboBox" name="comboBox">
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>打开</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Serial</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0,1">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QPushButton" name="pushButton_3">
                      <property name="text">
                       <string>清除</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPlainTextEdit" name="plainTextEdit"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>发送</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTextEdit" name="textEdit"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Chart</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_18" stretch="1,0,6,6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                     <string>GroupBox</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,0,2,0,2,0,2,1,1,1">
                       <item>
                        <widget class="QPushButton" name="pushButton_4">
                         <property name="text">
                          <string>AD Clear</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_5">
                         <property name="text">
                          <string>Bar Clear</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>MAX Index:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>MAX Value:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>T:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_9">
                         <property name="text">
                          <string>C:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_10">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_11">
                         <property name="text">
                          <string>T/C:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_12">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit">
                         <property name="text">
                          <string>1</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_6">
                         <property name="text">
                          <string>Calculate AD</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,10,0">
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_9"/>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,10,0">
                  <item>
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_15"/>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Chart2</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,0,9">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QGroupBox" name="groupBox_3">
                    <property name="title">
                     <string>GroupBox</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_21">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1,1,1,1,1,1,1">
                       <item>
                        <widget class="QPushButton" name="pushButton_7">
                         <property name="text">
                          <string>AD2 Clear</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_14">
                         <property name="text">
                          <string>Max Index:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_15">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_13">
                         <property name="text">
                          <string>Max Value:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_17">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_2"/>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_8">
                         <property name="text">
                          <string>Calculate AD2</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_16">
                         <property name="text">
                          <string>0</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_19">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,10,0">
                  <item>
                   <spacer name="verticalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_16"/>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>1</width>
                      <height>1</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>调试助手</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
